### PR TITLE
docs: Requirements for filenames and commands

### LINKED ISSUE #187: Update doc requirements doc on handling files and commands
Explain how filenames and commands should be rendered.

### PR DESCRIPTION
**Pull request by jodh-intel**
Explain how to handle filenames and command names in documentation.

Fixes #187.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
@klynnrif, @intelkevinputnam, @grahamwhaley - ptal.

File: Documentation-Requirements.md [hunk 1]
```diff
@@ -33,6 +34,25 @@ If there are multiple notes, bullets should be used:
...
 > - I am important point *n*.
 
+# Files and command names
+
+All filenames and command names should be rendered in a fixed-format font
+using backticks:
+
+- > Run the `foo` command to make it work.
+
+- > Modify the `bar` option in file `/etc/baz/baz.conf`.
```
Comment (grahamwhaley) [inline Documentation-Requirements.md]: Having these first two notes bulleted, but the next single pair not looks a little odd on rendering. I'd say either bullet all four, or none?

**Comment by jodh-intel** [comment]
Branch updated.

**Comment by nitkon** [comment]
lgtm :-)

File: Documentation-Requirements.md [hunk 2]
```diff
@@ -33,6 +34,25 @@ If there are multiple notes, bullets should be used:
...
+
+All filenames and command names should be rendered in a fixed-format font
+using backticks:
+
+> Run the `foo` command to make it work.
+
+> Modify the `bar` option in file `/etc/baz/baz.conf`.
+
+Any options that need to be specified to the command should also be rendered
+in the same manner:
```
Comment (klynnrif) [inline Documentation-Requirements.md]: Lines 46-47 suggested rewrite to stay active: 

Render any options that need to be specified to the command in the same manner:

**Comment by jodh-intel** [comment]
Thanks @klynnrif - branch updated.

**Pull request merged**